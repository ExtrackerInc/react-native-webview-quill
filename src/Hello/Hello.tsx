import * as React from 'react';
import { Text, View } from 'react-native';

export default class Hello extends React.Component {
  public render() {
    return (
      <View>
        <Text>Hello from the other side test!</Text>
      </View>
    );
  }
}
