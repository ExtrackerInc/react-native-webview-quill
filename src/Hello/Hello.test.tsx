import * as React from 'react';
import reactTestRenderer from 'react-test-renderer';
import Hello from './Hello';

describe('Hello', () => {
  it('should be on the other side', () => {
    const tree = reactTestRenderer.create(<Hello />).toJSON();
    expect(tree).toMatchSnapshot();
  });
});
