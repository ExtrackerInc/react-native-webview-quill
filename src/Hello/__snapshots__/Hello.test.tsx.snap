// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Hello should be on the other side 1`] = `
<View>
  <Text>
    Hello from the other side!
  </Text>
</View>
`;
